digdag-docs/src/metrics.md

```diff
@@ -0,0 +1,19 @@
+
+# Digdag metrics (experimental)
+
+Digdag support JMX and provided a few metrics. But it is not enough to monitor its performance and stability.
+To improve it, new metrics framework _digdag metrics_ has been introduced.
+These new metrics are provided via JMX.
+
+_Digdag metrics_ integrates micrometer
+There are meny metrics in _digdag merics_.
```

Comment:
meny -> many
merics -> metrics?

Reply:
@hiroyuki-sato Thanks for your pointing out. I will fix them later.